server/vote_endpoints.go

```diff
@@ -15,14 +15,40 @@
 package server
 
 import (
-	"net/http"
-
+	"encoding/json"
 	"github.com/rs/zerolog/log"
+	"io"
+	"net/http"
 
 	"github.com/RedHatInsights/insights-operator-utils/responses"
 	"github.com/RedHatInsights/insights-results-aggregator/types"
 )
 
+// readFeedbackRequestBody likes the rule for current user
+func (server *HTTPServer) readFeedbackRequestBody(writer http.ResponseWriter, request *http.Request) (types.FeedbackRequest, bool) {
+	var feedback types.FeedbackRequest
+
+	err := json.NewDecoder(request.Body).Decode(&feedback)
+	switch {
+	case err == io.EOF:
+		feedback.Message = ""
+		return feedback, true
+	case err != nil:
+		handleServerError(writer, err)
+		return feedback, false
+	}
+
+	if len(feedback.Message) > 250 {
```

Comment:
I would do it on storage side, but ok

Reply:
I agree, it should be done in the storage

Reply:
I don't think it should be done on storage side, because checking validity of request data is usually doing before any operations with storage.